### PR TITLE
chore: fix style for proper sizing in dialogs

### PR DESCRIPTION
**Pull request by Supamiu**
## Description

-----

-----

### CODE REVIEW

File: packages/ng/dialog/dialog.service.ts [hunk 1]
```diff
@@ -50,6 +52,10 @@ export class LuDialogService {
 			...(config.cdkConfigOverride || {}),
 		});
 
+		if (cdkRef.componentRef) {
+			this.#renderer.setStyle(cdkRef.componentRef.location.nativeElement, 'display', 'contents');
+		}
```
Comment (GuillaumeNury) [inline packages/ng/dialog/dialog.service.ts:57]: Can't we get a renderer using `cdkRef.componentRef.injector.get(Renderer2)` ?

Comment (Supamiu) [reply packages/ng/dialog/dialog.service.ts:57]: We can't because we're in a service so we don't have access to `Renderer2`

Comment (Supamiu) [reply packages/ng/dialog/dialog.service.ts:57]: Oh wait, I just realized you're right indeed 😄 I read that wrong and thought you meant to just use inject.

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2596/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13862/ui-diff)

**Review by jeremie-lucca — APPROVED** [review]

**Review by GuillaumeNury — APPROVED** [review]

**Pull request merged**